docs/root/intro/what_is_envoy.rst

```diff
@@ -73,10 +73,15 @@ and logging for MongoDB connections.
 NOSQL datastore. Envoy :ref:`supports <arch_overview_dynamo>` L7 sniffing and statistics production
 for DynamoDB connections.
 
-**Service discovery:** :ref:`Service discovery <arch_overview_service_discovery>` is a critical
-component of service oriented architectures. Envoy supports multiple service discovery methods
-including asynchronous DNS resolution and REST based lookup via a :ref:`service discovery service
-<arch_overview_service_discovery_types_sds>`.
+**Service discovery and dynamic configuration:** Envoy optionally consumes a layered set of
```

Comment:
@mattklein123 this and one other change are the only changes from #4842, addressing your comments.